native/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ cc_set_if_undefined(USE_JOB_SYSTEM_TBB       OFF)
 cc_set_if_undefined(USE_PHYSICS_PHYSX        OFF)
 cc_set_if_undefined(USE_MODULES              OFF)
 cc_set_if_undefined(USE_SERVER_MODE          OFF)
+cc_set_if_undefined(USE_CCACHE               ON)
```

Comment:
What's `CCACHE`?

Reply:
https://ccache.dev/  A tool to cache & speedup android build predure

Reply:
Any bad effect?

Reply:
Perhaps disk usage

Reply:
Does it cause any cache error?

Reply:
`ccache` is a mature tool supported by Android. The default configuration is changed to off, which can be enabled by developers themself. We need to add an option on the build panel.